content/20.treatments.md

```diff
@@ -505,6 +504,39 @@ Supplementation of vitamin D and maintaining a healthy diet for optimum vitamin
 This is particularly important considering ‘stay in place’ guidance has been implemented in many densely populated cities around the world.
 This measure is likely to limit people’s exposure to sunlight and thus reduce endogenous synthesis of vitamin D, potentially weakening the immune system and increasing the risk of COVID-19 infection.
 
+##### Probiotics
+
+Probiotics are “live microorganisms that, when administered in adequate amounts, confer a health benefit on the host” [@doi:10.1038/nrgastro.2014.66]. 
+As a result of the supposed modest benefits of probiotics against common viral infections and evidence that they can modulate the immune response [@doi:10.4082/kjfm.2013.34.1.2; @doi:10.2174/1381612824666180116163411], it is hypothesised that probiotics may have therapeutic benefits against SARS-CoV-2 [@doi:10.3389/fpubh.2020.00186].
+
+**Potential Mechanisms.**
+Probiotics and next generation probiotics, which are more akin to pharmacological grade supplements, have been associated with multiple potential beneficials effects for allergies, digestive tract disorders and even metabolic diseases through their anti-inflammatory and immunomodulatory effects [@doi:10.1038/nmicrobiol.2017.57; @doi:10.1093/advances/nmy063]. 
+However, the mechanisms by which probiotics affect these various conditions are diverse, heterogeneous, and strain specific [@doi:10.1093/advances/nmy063]. 
+Some of the beneficial effects of probiotics include reducing inflammation by promoting the expression of anti-inflammatory mediators, inhibiting toll-like receptors (TLR) 2 and 4, direct competition with pathogens, the synthesis of antimicrobial substances or other metabolites, improving intestinal barrier function, and/or favourably altering the gut microbiota and the brain-gut axis [@doi:10.1016/j.earlhumdev.2019.05.010; @doi:10.1093/advances/nmy063; @doi:10.1159/000342079]. 
```

Comment:
```suggestion
Some of the beneficial effects of probiotics include reducing inflammation by promoting the expression of anti-inflammatory mediators, inhibiting toll-like receptors (TLR) 2 and 4, direct competition with pathogens, the synthesis of antimicrobial substances or other metabolites, improving intestinal barrier function, and/or favorably altering the gut microbiota and the brain-gut axis [@doi:10.1016/j.earlhumdev.2019.05.010; @doi:10.1093/advances/nmy063; @doi:10.1159/000342079]. 
```
I am probably the worst American to have catching non-American English spellings because I went to school in England, but here goes!

Reply:
Apologies, my brain hasn't got used to the differences yet and in my stubbornness I refuse to change how I write the date and I will never understand Fahrenheit!